static/app/views/settings/components/dataScrubbing/index.tsx

```diff
@@ -14,7 +14,7 @@ import {Organization, Project} from 'sentry/types';
 import {defined} from 'sentry/utils';
 import useApi from 'sentry/utils/useApi';
 import {useNavigate} from 'sentry/utils/useNavigate';
-import {useRouteContext} from 'sentry/utils/useRouteContext';
+import {useParams} from 'sentry/utils/useParams';
```

Comment:
The mix of default and named exports in our routing utils is something we should probably address long term.

Reply:
cc @NisanthanNanthakumar